src/Uintra20/Content/uintra20/src/app/feature/shared/interfaces/IULink.ts

```diff
@@ -0,0 +1,9 @@
+export interface IUlinkWithTitle {
+  title: string;
+  link: IULink;
+}
+export interface IULink {
+  originalUrl: string;
+  baseUrl: string;
+  params: Array<object>;
+}
```

Comment:
after creating the new interface maybe need to replace old url in all components